src/Models/EventsModel.php

```
@@ -1,123 +1,150 @@
+                $row['Sector'] ?? [],
+                [['date' => $row['eventDate'], 'startTime' => null, 'endTime' => null]], 
+            );
+            $event->eventID = $row['id'];
+            $event->hoofdEventID = $row['hoofdEventID'];
+            $events[] = $event;
+        }
+    
+        return $events;
+    }
+
+    public static function delete($id){
```

PHPMD (2.15.0)

Avoid unused parameters such as '$id'. (undefined)

(UnusedFormalParameter)